Backpack-SwiftUI/Calendar/Classes/BPKCalendar.swift

```diff
@@ -40,6 +43,7 @@ public struct BPKCalendar<DayAccessoryView: View>: View {
     private var accessoryAction: ((Date) -> CalendarMonthAccessoryAction?)?
     private var initialMonthScroll: MonthScroll?
     private let monthHeaderDateFormatter: DateFormatter
+    private let calendarSelectionHandler: CalendarSelectionHandler
```

Comment:
I think this introduces a bit of inconsistency to the user of the component. as it makes unclear how to handle state.
For example, am I expected to change the state manually or is this just purely an event emissions thing?